lib/stats/stats-cluster.h

```diff
@@ -64,6 +62,17 @@ typedef enum _StatsClusterUnit
   SCU_GIB,
 } StatsClusterUnit;
 
+typedef enum _StatsClusterFormatting
+{
+  SCFOR_ABSOLUTE = 0,
+
+  /*
+   * Only applicable for time related counters.
+   * Results in a positive value for timestamps older than the time of query.
+   */
+  SCFOR_RELATIVE_TO_TIME_OF_QUERY,
+} StatsClusterFrameOfReference;
```

Comment:
the typedef is different to the struct name, but I think the "Formatting" name would be better, as I can imagine
multiple formatting qualifiers down the road, but I have a hard time imagining multiple frame of reference types.

Reply:
If we had some new formatting qualifier, that could be meaningful for both the absolute and relative frame of reference cases (like the frame of reference is meaningful for many of the unit enums, although not every one of them). If this was the case, the new formatting qualifier should be a separate enum (if we continue to implement this with enums and not switch to bitfields, like many other places in the code).

In my mind the unit is part of the formatting and the frame of reference is a different part of the formatting, so I did not want to give the new "logic" (relative/absolute) the name of the superset (formatting), and instead made a `formatting` struct, which consist of unit and frame of reference.

I think we are both trying to make this future proof, but in different ways. Maybe if we came up with some new qualifiers that could be implemented in the future, we could see this clearer. Either way, I am completely okay with changing the code if I did not convince you, as I don't think that I can come up with more arguments. :)